Tests/NetworkServiceTests/BodyEncoderTests/JSONBodyEncoderTests.swift

```diff
@@ -0,0 +1,36 @@
+import NetworkService
+import XCTest
+
+final class JSONBodyEncoderTests: XCTestCase {
+    let testURL = Bundle.main.bundleURL
+    var request: URLRequest!
+
+    override func setUp() {
+        super.setUp()
+        request = URLRequest(url: testURL)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        request = nil
+    }
+
+    func testEncodeExistsInAPI() throws {
+        XCTAssertNoThrow(try JSONBodyEncoder.encode(nil, into: &request))
+    }
+
+    func testEncodeAddedExpectedBodyParameters() throws {
+        let addingParameters = ["test": "param"]
+        let expectedData = try JSONEncoder().encode(addingParameters)
+
+        try JSONBodyEncoder.encode(addingParameters, into: &request)
+
+        let existingBody = request.httpBody
+        XCTAssertEqual(expectedData, existingBody)
+    }
```

Comment:
```suggestion
            // Given
        let addingParameters = ["test": "param"]
        let expectedData = try JSONEncoder().encode(addingParameters)
        
        // When
        try JSONBodyEncoder.encode(addingParameters, into: &request)

        // Then
        XCTAssertEqual(expectedData, request.httpBody)
    }
   ```
I think this version is more readable.

Reply:
Agree, fixed with https://github.com/ios-course/simple-network-service/pull/13/commits/d63c1f24dc3b58f1adec0e4bcfd6400e46f5c258

Reply:
Are you sure that file was updated according to suggestion?

Reply:
My bad, I forgot about Given-When-Then. Fixed with https://github.com/ios-course/simple-network-service/pull/13/commits/92115fd5f550248944ac406b0dfded694803645a